src/NuGet.Clients/NuGet.PackageManagement.UI/DisplayVersion.cs

```diff
@@ -48,22 +51,28 @@ public class DisplayVersion
             Version = range.MinVersion;
             IsCurrentInstalled = isCurrentInstalled;
             AutoReferenced = autoReferenced;
+            IsDeprecated = isDeprecated;
 
             // Display a single version if the range is locked
-            if (range.HasLowerAndUpperBounds && range.MinVersion == range.MaxVersion)
-            {
-                var formattedVersionString = Version.ToString(versionFormat, VersionFormatter.Instance);
+            // If range is unlocked, display it and use the original value for floating ranges
+            var versionString = range.HasLowerAndUpperBounds && range.MinVersion == range.MaxVersion ?
+                Version.ToString(versionFormat, VersionFormatter.Instance)
+                : Range.OriginalString;
 
-                _toString = string.IsNullOrEmpty(AdditionalInfo) ?
-                    formattedVersionString :
-                    AdditionalInfo + " " + formattedVersionString;
+            _toString = "";
+            if (!string.IsNullOrEmpty(AdditionalInfo))
+            {
+                _toString += AdditionalInfo + " ";
             }
-            else
+
+            _toString += versionString;
+
+            if (IsDeprecated)
             {
-                // Display the range, use the original value for floating ranges
-                _toString = string.IsNullOrEmpty(AdditionalInfo) ?
-                    Range.OriginalString :
-                    AdditionalInfo + " " + Range.OriginalString;
+                _toString += string.Format(
+                    CultureInfo.CurrentCulture,
+                    "    ({0})",
```

Comment:
Feels like there's more than one space. How many should there be?

Reply:
I had one space at first, and then @rrelyea suggested adding multiple spaces and we thought it looked better. There's 4 I think? Unfortunately we're heavily restricted with how the text is formatted and displayed due to the nature of ComboBox and felt this was the easiest stopgap.

Reply:
Best effort :)